examples/config/cosmodc2_config.yml

```diff
@@ -85,13 +85,27 @@ TXSourceSelector:
     i_hi_cut: 19.9
     r_i_cut: 2.0
     # may also need one for r_cpar_cut
+    true_z: True
+    shear_prefix: mcal_
 
 TXRandomCat:
     chunk_rows: 100000
     density: 10 # gals per sq arcmin
 
+TXJackknifeCenters:
+    npatch: 40
+
+TXDiagnosticPlots:
+    psf_prefix: mcal_psf_
```

Comment:
We weren't using the mcal PSFs because they are rounded - i.e. all the ellipticities are close to zero, whatever the actual PSF.

Reply:
I only added this because it was raising an error otherwise.. I'll remove it for the moment though.